metadata/config/config_test.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2023 VMware, Inc.
+//
+// This product is licensed to you under the BSD-2 license (the "License").
+// You may not use this product except in compliance with the BSD-2 License.
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to
+// the terms and conditions of the subcomponent's license, as noted in the
+// LICENSE file.
+//
+// SPDX-License-Identifier: BSD-2-Clause
+
+package config
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewUpdaterConfig(t *testing.T) {
+
+	remoteURL := "somepath"
+	rootBytes := []byte("somerootbytes")
+
+	updaterConfig, err := New(remoteURL, rootBytes)
+
+	assert.Nil(t, err)
+
```

Comment:
It would be nice if we assert the `updaterConfig` value and the rest of the struct fields (i.e. their default values)